packs/data/classfeatures.db/surprise-attack.json

```diff
@@ -21,7 +21,26 @@
         "prerequisites": {
             "value": []
         },
-        "rules": [],
+        "rules": [
+            {
+                "key": "EphemeralEffect",
+                "predicate": [
+                    "encounter:round:1",
+                    {
+                        "lt": [
+                            "self:participant:initiative:rank",
+                            "target:participant:initiative:rank"
+                        ]
+                    }
+                ],
```

Comment:
`...if you roll Deception or Stealth for initiative...` Need to account for this too.

Reply:
whoops, right!